.travis.yml

```diff
@@ -45,5 +47,16 @@ script: |
     -configuration Debug ONLY_ACTIVE_ARCH=YES ENABLE_TESTABILITY=YES CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
   # Lint - Add `--strict` to fail for warning and violation
   swiftlint lint --reporter json
+  # Making document for core SDK
+  - make docs
 
 after_success: slather
+    - if [ $TRAVIS_BRANCH = 'master' ] && [ $TRAVIS_PULL_REQUEST = 'false' ]; then sh ./scripts/travisPublishDocs.sh; fi
+deploy:
+    provider: pages
+    skip-cleanup: true
+    github-token: $CI_USER_TOKEN  # Set in the settings page of your repository, as a secure variable
+    local-dir: docs
+    target-branch: gh-pages
+    on:
+       branch: master
```

Comment:
If you are using ./scripts/travisPublishDocs.sh; to publish documents then why do you need this deploy section?

Reply:
Updated